Update docs related to the search bar (#24515)
gravitational/teleport: docs/pages/connect-your-client/teleport-connect.mdx

  Teleport clusters as well as log in or out of the clusters.
- The **connection list** (the top left) showing recent connections, allowing you to seamlessly switch
  between them.
- The **command bar** (in the middle), which launches any command you input in a new terminal tab.
- The **cluster selector** (to the left of the command bar), which shows up only if you have set up
- The **search bar** (in the middle), which allows you to search for resources across clusters.
- The **cluster selector** (to the left of the search bar), which shows up only if you have set up
  Trusted Clusters and there are leaf clusters connected to the root cluster. It lets you browse

>   leaf cluster resources. It also changes which cluster the commands in the command bar are going to target.

Reviewer: Let's keep this:

> Also, the "Open new terminal" action will bound new terminal tabs to the selected cluster.

> - The **more options menu** (to the left of the profile selector), containing options such as

Reviewer: idk, I feel like we cannot refer to it as "the more options menu" because it looks terrible in text. For example, when talking about the cluster selector I'd like to add this text:

> It also changes which cluster the local terminal tabs opened through "Open new terminal" from the more options menu are going to be bound to.

I searched through some old Apple docs, e.g. https://web.archive.org/web/20150818000225/https://support.apple.com/en-us/HT204839, and it looks like in their UI the More Options icon is always referred to in the context of another specific part of the UI.

FWIW, these are not options, each item refers to an action. But even if we call it "more actions" then "more actions menu" still looks bad.

What about renaming it to something like "additional actions menu"? Instead of saying "click the More Options icon" we'd say "open the additional actions menu" and then the user can refer to this section of the docs to understand what the "additional actions menu" is.

Author: I don't have a strong opinion on it, but I think this rename makes sense 👍

>   opening a config file or creating an access request (if you are connected to an Enterprise cluster).

Reviewer: I'm not sure about the access request part, what bothers me is that additional explanation in parentheses. We're in the section which explains how the UI looks and that conditional in the parens puts too much focus on this additional context of enterprise clusters. It's a little bit too wordy in my opinion.

OTOH there's really no other action we could mention as "Open new terminal" is mentioned above and the other two actions are macOS only.

What do you think about this then?

```suggestion
  opening a config file or creating an access request in an Enterprise cluster.
```

Author: Thanks for this small improvement, it indeed reads better 👍
